v2/pkg/apis/kubeflow/v2beta1/types.go

```diff
@@ -55,6 +55,9 @@ type MPIJobSpec struct {
 	// +kubebuilder:default:="/root/.ssh"
 	SSHAuthMountPath string `json:"sshAuthMountPath,omitempty"`
 
+	// Spawn launcher after all workers are in Ready state if true
```

Comment:
The comment needs to start with the parameter name `WaitForWorkers`; something like:

```suggestion
	// WaitForWorkers if true, the launcher is created only after all workers are in Ready state
```